vaccine_feed_ingest/utils/normalize.py

```diff
@@ -7,142 +7,118 @@
 import url_normalize
 from vaccine_feed_ingest_schema.location import VaccineProvider
 
+VACCINE_PROVIDER_REGEXES = {
+    VaccineProvider.ALBERTSONS: {
```

Comment:
I am concerned about this data model. This process combines brands that are run by the same corporation into the same id space. Are you sure that is the way it works in practice?

By this I mean, is Albertsons #5301 and Safeway #5301 the same store, or are they different stores?

Reply:
What do you think of extracting the provider, brand, and id and creating an ID from those parts to ensure uniqueness e.g. `albertsons:osco_5301` or `albertsons_osco:5301`

Reply:
I don't know enough about either our front-end, or how the data is used to truly weigh in, but after reviewing:
https://www.cdc.gov/vaccines/covid-19/retail-pharmacy-program/participating-pharmacies.html - my suggestion is that we enhance the schema to explicitly represent brands, providers, and the relationship between them.

To be more specific:

* The `VaccineProvider` enum hereafter represents individual brands (e.g., Big Y Pharmacy, Brookshires Pharmacy, etc)
* A new enum, say: `VaccineProviderParent` (with values like Topco, Kroger, etc) represents businesses that have one or more `VaccineProvider` subsidiaries
* Add a `getParent()` helper method to  the `VaccineProvider` enum and implement it appropriately to return an `Optional[VaccineProviderParent]`.
* Modify `provider_id_from_name()` to return either a `VaccineProvider` or `VaccineProviderParent` (use the `Union` type).

Reply:
Explicitly representing brands, parents, and their relationship in the schema will make it easier to build and maintain richer frontends showing users these relationships (if necessary).

Reply:
Tagging @nschiefer to weigh in since he created #254

Reply:
Per discussion on Discord, the above proposal is fine, but the adding of the parent enum etc can be done later. For now, this PR will proceed with the assumption that `VaccineProvider` represents individual brands.

Reply:
Done.